### DISCUSSION TITLE
遺失中文字

### DISCUSSION THREAD

**Discussion by jkl54555** [jkl54555/Unity-TMPro-Chinese-Font#2] [General]
中文字遺失或其他問題
可以在此討論區告知

**Comment by louise-bwsx** [comment]
嘿嘿嘿 我這裡有個16291個字 的字庫 已經把你的字全部包含進去了 有全形半形 注音 簡體 日文 特殊符號
還有一個我用來檢查所有字有沒有重複的工具 因為不給我上傳cs檔我給他改txt
[AllCharacters.txt](https://github.com/jkl54555/Unity-TMPro-Chinese-Font/files/13695397/AllCharacters.txt)
[StringFilter.txt](https://github.com/jkl54555/Unity-TMPro-Chinese-Font/files/13695414/StringFilter.txt)

**Reply by ykyyykk** [reply]
新增「裏」
[AllCharacters.txt](https://github.com/user-attachments/files/17564172/AllCharacters.txt)

**Reply by jkl54555** [reply]
> 新增「裏」 [AllCharacters.txt](https://github.com/user-attachments/files/17564172/AllCharacters.txt)

[louise-bwsx](https://github.com/jkl54555/Unity-TMPro-Chinese-Font/discussions/2#discussioncomment-7875455)提供的[字庫](https://github.com/jkl54555/Unity-TMPro-Chinese-Font/files/13695397/AllCharacters.txt)是有「裏」的
，我這兩天合併一下

**Reply by jkl54555** [reply]
已更新!
下載連結: [TMProChinese.v1.2.unitypackage](https://github.com/jkl54555/Unity-TMPro-Chinese-Font/releases/download/v1.2/TMProChinese.v1.2.unitypackage)

**Comment by alanwnl** [comment]
「裏」字好想未能顯示
<img width="733" alt="Screenshot 2024-10-29 at 10 52 51 AM" src="https://github.com/user-attachments/assets/b30f5292-b4d3-49a3-a8c8-55b21d433670">

**Reply by ykyyykk** [reply]
謝謝告知 我已經新增好了 你可以手動新增 或是 下載我新增好 裏 字的檔案做更換

**Reply by jkl54555** [reply]
謝謝告知，已更新。
下載連結: [TMProChinese.v1.2.unitypackage](https://github.com/jkl54555/Unity-TMPro-Chinese-Font/releases/download/v1.2/TMProChinese.v1.2.unitypackage)
或從[Releases](https://github.com/jkl54555/Unity-TMPro-Chinese-Font/releases)尋找最新包
